### ISSUE TITLE
HTTP webhook action is in running state instead of success.

### ISSUE THREAD

**Issue by vgouth** [Azure/logicapps#1230]
### Describe the Bug with repro steps 

1\.Create a Stateful workflow in the Standard Logic App.

 2\.Open designer. And Add http request trigger and HTTP webhook action.

 3\.Use Method as Post , URL as Unique URL from Request Bin, Body as Callback URL token from dynamic token picker for all Subscribe and Unsubscribe parameters.

 4\.Save and Run the workflow.

**Expected:** Workflow Run should successful without any issues

**Actual:** Workflow run is in running state for very long period of time.

**Note:** Now the request bin URL is managed by pipedream and we have very limited access to use Unique URLs

### What type of Logic App Is this happening in? 

Standard (Portal)

### Which operating system are you using? 

Windows

### Are you using new designer or old designer 

New Designer

### Did you refer to the TSG before filing this issue? [https://aka.ms/lauxtsg](<https://aka.ms/lauxtsg>) 

Yes

### Workflow JSON 

*No response*

### Screenshots or Videos 

![Image](<https://github.com/user-attachments/assets/dacf45aa-5088-4317-b77d-f35ce832160d>)

![Image](<https://github.com/user-attachments/assets/1b3ab4fb-7fbd-4133-b77a-98a86028a6ef>)

### Browser 

Edge

### Additional context 

**Version :** 2\.41028.1.3

**Comment by vgouth** [comment]
@hartra344 HAR file is not accepting in email due to some company policy issues. I have attached the HAR file in this bug itself.

**Comment by hartra344** [comment]
Reach out about how to share HAR files in the future, please do not attach them to GitHub bugs. They can contain secrets and this issue can be seen by the public.

**Comment by vgouth** [comment]
Issue still repro in latest test pass. Attached sanitized HAR file over mail.

**Version:**0.2022.1.796

![Image](https://github.com/user-attachments/assets/a8f4a2a2-9eee-43a8-b863-502c2d203a29)

**Comment by vgouth** [comment]
Issue repro in latest canary test pass 
**Version:** 0.2022.1.811

![Image](https://github.com/user-attachments/assets/e7bcaeb9-111b-4611-a304-4c711bad7d33)

**Comment by vgouth** [comment]
Issue repro in latest canary test pass
Version: 0.2022.1.811

**Comment by vgouth** [comment]
Issue repro in stage environment
**Version:** 2.41231.2.5

![Image](https://github.com/user-attachments/assets/995c7e04-a9e0-4b60-9c0a-60142dfa2725)

**Comment by rarayudu** [comment]
Are you invoking the callback url that is sent as part of the Http Webhook subscribe. As you can see in the outputs, the webhook action is waiting for the invocation on the callback.

**Comment by vgouth** [comment]
@rarayudu yes, we are invoking the callback url that is sent as a part of http webhook subscribe and unsubscribe.

**Comment by vgouth** [comment]
Issue repro in stage environment
**Version:** 2.50203.1.1

![Image](https://github.com/user-attachments/assets/537bc411-e24d-4b37-852f-e192d9a579cb)

![Image](https://github.com/user-attachments/assets/73bac5d1-e19c-4fff-80f3-925a12b5092e)

**Comment by vgouth** [comment]
Issue **not** repro when we use **hookdeck** website as an alternative of pipedream(request bin URL). Probably it could be an issue from pipedream website. Closing this issue for now.
**Version:** 2.50203.1.3
![Image](https://github.com/user-attachments/assets/37c8e1ce-be5a-4f70-837b-83c93296fb61)

**Comment by vgouth** [comment]
Issue repro in the latest staging test pass when we use hookdeck website as an alternative of pipedream(request bin URL).
**Version:** 2.50224.1.3

![Image](https://github.com/user-attachments/assets/562905a2-f41c-4643-89ea-a80f784c3d1d)

**Comment by vgouth** [comment]
Issue repro in latest staging test pass

**Version:** 2.50324.1.6

![Image](https://github.com/user-attachments/assets/7b2025f8-853c-45b5-b9e5-7de7c3606907)

**Comment by vgouth** [comment]
Issue repro in latest staging test pass
Version: 2.50425.1.6

![Image](https://github.com/user-attachments/assets/7803c314-5154-4278-970e-a9d04fa236fc)

**Comment by vgouth** [comment]
Issue repro in latest staging test pass
**Version:** 2.50818.1.2

<img width="1343" height="551" alt="Image" src="https://github.com/user-attachments/assets/0511be65-9208-467b-801b-dc80ddb5f840" />

**Comment by v-ksruthi** [comment]
Issue repro in latest Stage environment
Version:2.50912.1.1

<img width="1064" height="561" alt="Image" src="https://github.com/user-attachments/assets/54b6f5ca-a6b5-423e-bcd5-bc28ed054820" />

<img width="1083" height="223" alt="Image" src="https://github.com/user-attachments/assets/b711a30d-c61c-491b-b8ff-48942d9b977e" />

**Comment by v-ksruthi** [comment]
Issue repro in latest staging test pass
Version:2.51008.1.2

<img width="1082" height="606" alt="Image" src="https://github.com/user-attachments/assets/1b12a033-7674-4621-abfd-72d903a55e46" />

<img width="1356" height="596" alt="Image" src="https://github.com/user-attachments/assets/5a52a0a0-c923-43f2-9495-759877151ce8" />

**Comment by v-ksruthi** [comment]
Issue repro in latest stage environment
Version:2.60223.1.3

<img width="1168" height="559" alt="Image" src="https://github.com/user-attachments/assets/93369f5b-8636-4cde-bc57-763d7457c0ca" />

<img width="1364" height="623" alt="Image" src="https://github.com/user-attachments/assets/e46e0418-4410-4811-a807-15c072867f75" />

**Comment by v-ksruthi** [comment]
Issue repro in latest vscode request in Bundle artifact
VSIX Version:5.273.0
Bundle Version:1.165.8

<img width="1032" height="678" alt="Image" src="https://github.com/user-attachments/assets/ed9ef58c-22d5-468d-87c1-ea5b03259a9e" />

**Comment by v-ksruthi** [comment]
Issue repro in latest canary testpass
Version:2.0.20260415.3

<img width="1360" height="579" alt="Image" src="https://github.com/user-attachments/assets/149a25f2-dd48-4dae-99b8-2b6771e4e3ac" />

**Comment by v-ksruthi** [comment]
Issue repro in latest stage request
Version;2.60514.1.5

<img width="1279" height="597" alt="Image" src="https://github.com/user-attachments/assets/c2707530-7c24-49cf-9c67-434d9710c246" />

**Comment by v-ksruthi** [comment]
Issue repro in latest stage request
Version: 2.60615.1.1

<img width="1365" height="602" alt="Image" src="https://github.com/user-attachments/assets/7b733f0a-22f0-42f9-b149-0c1d50e9424d" />
